### DISCUSSION TITLE
How to export projects list

### DISCUSSION THREAD

**Discussion by kinh-nguyen** [alefragnani/vscode-project-manager#690] [Q&A] [answered]
When switching from VSCode to VSCodium for example, the plugin works fine but how can we export and use the saved projects? 
Thanks!

**Comment by alefragnani** [comment, marked as answer]
Hi @kklot ,

There is no export command, but you can use the `Project Manager: Edit Projects` command instead, which will open the `project.json` file where the projects are stored. Then, you can simply copy its content and paste it in VSCodium, where you also opened `projects.json` file using the same command.

Hope this helps
